src/components/MiniMap.vue

```diff
@@ -1,22 +1,45 @@
 <template>
-  <div class="mini-paper-container" @click="movePaper" :class="isOpen ? 'opened' : 'closed'">
-    <div ref="miniPaper" class="mini-paper" />
+  <div class="mini-map-btn">
+    <b-button
+      class="btn btn-sm ml-auto"
+      data-test="mini-map-btn"
+      @click="isOpen = !isOpen"
+      v-b-tooltip.hover
+      :title="isOpen ? $t('Hide Mini-Map') : $t('Show Mini-Map')"
+    >
+      <font-awesome-icon :icon="miniMapOpen ? minusIcon : mapIcon" />
```

Comment:
I'm seeing an error that miniMapOpen is not defined.

Reply:
Thanks, looks like I missed that reference in the refactor.